src/snowflake/connector/s3_storage_client.py

```diff
@@ -80,6 +80,7 @@ def __init__(
             )
         )
         self.use_s3_regional_url = use_s3_regional_url
+        self.location_type = stage_info["locationType"]
```

Comment:
is this `locationType` information guaranteed to be provided in the stage_info?
shall we use stage_info.get("locationType")?

Reply:
As I talked with people from storage team, this is a stable way to decide where the endpoint is, will use get()